docs/_articles/configuration.md

```diff
@@ -73,6 +73,24 @@ Configure the Sentry SDK client to send error and performance reports to a servi
 - SENTRY_TRACES_SAMPLE_RATE - Defaults to `0.05`. Set to the percent of requests to send performance traces. Or `0.0` for disabled.
 - RAVEN_JS_DSN - Disabled by default. Same as SENTRY_DSN but for the JavaScript frontend.
 
+### Single Sign On
+
+Variables map to [django-allauth configuration](https://django-allauth.readthedocs.io/en/latest/configuration.html). Supported options:
+
+- ACCOUNT_EMAIL_VERIFICATION
```

Comment:
Should we tell we use `mandatory` as default?

Reply:
Updated to specify the default, which is not the upstream default!